tests/govtool-frontend/playwright/global-setup.ts

```diff
@@ -5,6 +5,9 @@ import { loadAmountFromFaucet } from "@services/faucetService";
 import kuberService from "@services/kuberService";
 import walletManager from "lib/walletManager";
 
+const REGISTER_DREP_WALLETS = 9;
+const DREP_WALLETS = 9;
```

Comment:
Rename variables to have `COUNT` somewhere

Reply:
Done